stdlib/distutils/fancy_getopt.pyi

```diff
@@ -1,10 +1,11 @@
 from collections.abc import Iterable, Mapping
+from getopt import _SliceableT, _StrSequenceT_co
 from re import Pattern
 from typing import Any, Final, overload
 from typing_extensions import TypeAlias
 
 _Option: TypeAlias = tuple[str, str | None, str]
-_GR: TypeAlias = tuple[list[str], OptionDummy]
+_GR: TypeAlias = tuple[_StrSequenceT_co, OptionDummy]
```

Comment:
I don't think this type alias is really pulling its weight. Inlining might be clearer.